iteration/iteration.go

```diff
@@ -60,13 +64,41 @@ type GormIterationRepository struct {
 	db *gorm.DB
 }
 
+// ConvertToLtreeFormat returns LTREE valid string
+func ConvertToLtreeFormat(uuid string) string {
+	//Ltree allows only "_" as a special character.
+	return strings.Replace(uuid, "-", "_", -1)
+}
+
+// ConvertFromLtreeFormat returns UUID in form of string
+func ConvertFromLtreeFormat(uuid string) string {
+	// Ltree allows only "_" as a special character.
+	converted := strings.Replace(uuid, "_", "-", -1)
+	converted = strings.Replace(converted, PathSepInDatabase, PathSepInService, -1)
+	return converted
+}
+
+// LoadMultiple returns multiple instances of iteration.Iteration
+func (m *GormIterationRepository) LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]*Iteration, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "Iteration", "getmultiple"}, time.Now())
```

Comment:
yes yes done in https://github.com/almighty/almighty-core/pull/713/commits/03c25970ecb85bc623af2f97ae5023be16fb2d73  :)